picasso/src/main/java/com/squareup/picasso/RemoteViewsAction.java

```diff
@@ -27,25 +27,41 @@
 abstract class RemoteViewsAction extends Action<RemoteViewsAction.RemoteViewsTarget> {
   final RemoteViews remoteViews;
   final int viewId;
+  Callback callback;
```

Comment:
I couldn't tell if `@Nullable` was appropriate to start adding, so I omitted it.